website/src/components/lineage/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+/** As a private package, not every devleoper will have access to this repo. */
+const DagExports = require('@dbt-labs/react-dbt-dag');
+
+export const Lineage = ({ dagNodes, currentNodeId }) => {
+
+    if (!DagExports) {
+        return <div>Lineage Not Available!</div>
+    }
```

Comment:
I haven't tested this out thoroughly, but @jgiuffrida claimed something like this pattern would allow other developers to still access this in development while not having access to the private repo